src/DynamoCore/Engine/EngineController.cs

```diff
@@ -302,25 +258,6 @@ internal List<Guid> PreviewGraphSyncData(IEnumerable<NodeModel> updatedNodes, bo
             return previewGraphQueue.Dequeue();
         }
 
-        /// <summary>
-        /// Returns true if there are graph sync data in the queue waiting to be executed.
-        /// </summary>
-        /// <returns></returns>
-        public bool HasPendingGraphSyncData
```

Comment:
I don't expect that there would be any client using this public method, but this may break binary compatibility.